src/kmergraph.cpp

```diff
@@ -347,37 +578,38 @@ float KmerGraph::find_max_path(std::vector<KmerNodePtr> &maxpath, const uint32_t
     assert(num_reads > 0 || assert_msg("num_reads was not set in kmergraph"));
 
     // need to catch if thesh not set too...
-    check();
+    kmer_prg->check();
 
     // also check not all 0 covgs
     auto coverages_all_zero = coverage_is_zeroes(sample_id);
     if (coverages_all_zero)
         return std::numeric_limits<float>::lowest();
 
     // create vectors to hold the intermediate values
-    std::vector<float> M(nodes.size(), 0); // max log prob pf paths from pos i to end of graph
-    std::vector<int> len(nodes.size(), 0); // length of max log path from pos i to end of graph
-    std::vector<uint32_t> prev(nodes.size(), nodes.size() - 1); // prev node along path
+    std::vector<float> M(kmer_prg->nodes.size(), 0); // max log prob pf paths from pos i to end of graph
```

Comment:
Rename `M`

Reply:
I think @rmcolq will know how to rename `M` better

Reply:
updated by commit bec1ff33d23fb4178752ee6a586f07ccd6d9a0d0